metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py

```diff
@@ -85,6 +85,12 @@ class AzureADConfig(StatefulIngestionConfigBase, DatasetSourceConfigMixin):
         description="[Microsoft Graph API endpoint](https://docs.microsoft.com/en-us/graph/use-the-api)",
     )
 
+    # Optional: Specify the version of Azure AD endpoint
+    endpoint_version: str = Field(
```

Comment:
is there a way we can auto-detect this?

Reply:
if not, we should tighten the validation to only allow "v1" and "v2" and throw an error on all other strings